### DISCUSSION TITLE
New Technique Ideas

### DISCUSSION THREAD

**Discussion by openrec0n** [vectra-ai-research/Halberd#23] [Ideas]
Improving testing coverage is a primary goal. Share/link new techniques that should be added to the tool and will help others .

**Comment by jesayafn** [comment]
Following issue #30, making support wider to GCP or other cloud providers is a must. As a starter, we can start a technique that is similar to AWS or GCP, such as exposing a cloud storage bucket or listing all available service accounts. Using JSON file as an authentication method to establish access.

**Reply by openrec0n** [reply]
#37 Adds initial support for GCP in Halberd. @jesayafn great job adding this
